indico/cli/i18n.py

```diff
@@ -335,6 +335,68 @@ def remove_empty_pot_files(path: Path, python=False, javascript=False, react=Fal
         _get_messages_react_pot(path).unlink()
 
 
+# Merge after generating all the .pot files
+def merge_pot_files(output_file, *input_files):
+    """Merge multiple POT files into a single POT file."""
+    merged_catalog = None
+
+    print('\n🗨  Merging POT files...\n')
```

Comment:
We normally use `click.echo` or `click.secho` for CLI output, let's use it here as well (and no need for so many newlines :D):

```suggestion
    click.secho('🗨  Merging POT files...', fg='yellow', bold=True)
```

Reply:
Thanks! I removed this particular print but I'll make a green line for the 'finish' print